docs/hub/datasets-image.md

```diff
@@ -0,0 +1,123 @@
+# Image Dataset
+
+This guide will show you how to configure your dataset repository with image files.
+
+A dataset with a supported structure and [file formats](./datasets-adding#file-formats) automatically has a Dataset Viewer on its dataset page on the Hub. Any additional information about your dataset - such as captioning or object detection - is automatically loaded as long as you include this information in a metadata file (`metadata.csv`/`metadata.jsonl`).
+
+## Only images
+
+If your dataset only consists of one column with images, you can use a simple setup: store your image files at the root:
+
+```
+1.jpg
+2.jpg
+3.jpg
+4.jpg
+```
+
+or using subdirectories:
+
+```
+images/1.jpg
+images/2.jpg
+images/3.jpg
+images/4.jpg
+```
```

Comment:
````suggestion
If your dataset only consists of one column with images, you can simply store your image files at the root:

```
1.jpg
2.jpg
3.jpg
4.jpg
```

or use subdirectories:

```
images/1.jpg
images/2.jpg
images/3.jpg
images/4.jpg
```
````

Maybe it's also better to use the `tree` command's formatting for these file structures.